prototype/index.html

```diff
@@ -106,8 +106,13 @@
             document.querySelector("#searchResults").innerHTML = "";
             
             for (let entry of entries) {
```

Comment:
Instead of now iteration 4 times over the search results I would propose:

```suggestion
            for (let icon of result.data.search) {
            if (versionType === 'Free' && !isAvailableInFree(icon)) {
                console.log(`Skipped ${icon.id}`)
            }
            
            const displayElement = document.createElement('i');
            displaceElement.classList.add(isBrand(icon) ? 'fab' : 'fas');
            ...
        }
        
        function isAvailableInFree(icon) {
            // Only brands and solids are available in free.
            return icon.membership.free.includes('solid') || icon.membership.free.includes('brands');
        }
        
        function isBrand(icon) {
            return icon.membership[versionType.toLowerCase()].includes('brands');
        }
```